files/root/grafana_tools/ldap_sync/new_team_member_template.json

```diff
@@ -0,0 +1,3 @@
+{
+  "userId": NewUserIDGoesHere
```

Comment:
Does this need double-quotes?

Reply:
Again, these files come from another git repo. I don't want to start editing or recreating the files.